content/hi/docs/tutorials/hello-minikube.md

```diff
@@ -0,0 +1,306 @@
+---
+title: हेलो मिनीक्यूब
+content_type: tutorial
+weight: 5
+menu:
+  main:
+    title: "शुरू करते हैं"
+    weight: 10
+    post: >
+      <p>तो क्या आप तैयार हैं? एक सामान्य ऐप चलने के लिए एक साधारण कुबेरनेट्स क्लस्टर बनाएं।</p>
+card:
+  name: tutorials
+  weight: 10
+---
+
+<!-- overview -->
+
+यह ट्यूटोरियल आपको मिनिक्यूब और काटाकोडा का उपयोग करते हुए 
+कुबेरनेट्स पर एक साधारण ऐप चलाने का तरीका दिखाता है।
+काटाकोडा आपके ब्राउज़र पर मुफ़्त कुबेरनेट्स वातावरण प्रदान करता है।
+
+{{< note >}}
+यदि आपने अपने स्थानीय सिस्टम पर मिनीक्यूब स्थापित किया है तो आप इस ट्यूटोरियल का अनुसरण कर सकते हैं।
+स्थापाना निर्देश के लिए [मिनीक्यूब पृष्ट](https://minikube.sigs.k8s.io/docs/start/) देखें। 
+{{< /note >}}
+
+## {{% heading "objectives" %}}
+
+* मिनीक्यूब में एक नमूना एप्लीकेशन डेप्लॉय करें।
+* ऐप को चलाएं।
```

Comment:
ऐप चलाएं। should be enough

Reply:
ऐप को चलाएं। sounds in sync with "run the app", (putting some weight on the word app is a good idea i guess?)
ऐप चलाएं is like saying "run app".
The English version also uses "run the app".

I found this really good article explaining को's grammer :)
https://hindilanguage.info/hindi-grammar/postpositions/ko/